cfme/common/provider.py

```diff
@@ -728,3 +729,22 @@ def cleanup_vm(vm_name, provider):
     except:
         # The mgmt_sys classes raise Exception :\
         logger.warning('Failed to clean up VM %s on provider %s', vm_name, provider.key)
+
+
+def get_certificate(provider, sec_protocol=None, cert_source=None):
```

Comment:
I went back and forth beween different classes with this one. This will be used in the future with VM providers as well. Not only Containers Providers. @jkrocil can you advise please..